Tests/Tracking/migration/DataPipelineMigrationAssistantTests.swift

```diff
@@ -0,0 +1,76 @@
+@testable import CioDataPipelines
+@testable import CioInternalCommon
+import Foundation
+@testable import Segment
+import SharedTests
+import XCTest
+
+class DataPipelineMigrationAssistantTests: UnitTest {
```

Comment:
I just moved them from [`CustomerIOImplementationTests`](https://github.com/customerio/customerio-ios/blob/main-replica-for-cdp/Tests/DataPipeline/DataPipelineInteractionTests.swift#L413-L452) to a separate file.